tests/sklearn/test_metrics.py

```diff
@@ -1,5 +1,7 @@
 import numpy as np
+import pytest
 from sklearn.linear_model import LogisticRegression
+from sklearn.utils.testing import assert_almost_equal
```

Comment:
Can we change this to `from numpy.testing import assert_almost_equal`? That's where sklearn get it and I was having trouble finding the documentation at first.